charts/graph-kubernetes/values.yaml

```diff
@@ -2,9 +2,9 @@
 ########### General ############
 ################################
 image:
-  repository: jupiterone/graph-kubernetes
+  repository: https://ghcr.io/jupiterone/graph-kubernetes
```

Comment:
I don't think we should specify a protocol for the url here.  This is how github suggests forming a docker pull command: `docker pull ghcr.io/jupiterone/graph-github:5.10.11`.

Reply:
I tried without at first and it failed on me. After adding the protocol it started working